api_v3/lib/types/enums/KalturaBaseEntryCloneOptions.php

```diff
@@ -0,0 +1,8 @@
+<?php
+/**
+ * @package api
+ * @subpackage enum
+ */
+class KalturaBaseEntryCloneOptions extends KalturaEnum implements BaseEntryCloneOptions

```

Comment:
KalturaBaseEntryCloneOptions should extend KalturaDynamicEnum in order to enable plugins to extend it.